src/address.rs

```diff
@@ -150,6 +151,17 @@ impl<A, Rc: RefCounter> Address<A, Rc> {
         SendFuture::sending_named(tx, rx)
     }
 
+    /// Send a message to all actors on this address. The message will be received once by each
+    /// actor. Note that currently there is no message cap on the broadcast channel (it is unbounded).
+    pub fn broadcast<M>(&mut self, msg: M) -> Result<(), Disconnected>
```

Comment:
I think this should also be supported on `MessageChannel`.

A public test would also be nice!

Reply:
Good point, I forgot about that! A test is good, and I also forgot to expose the priority of the broadcast here